src/sql/base/browser/ui/taskbar/actionbar.ts

```diff
@@ -238,6 +238,10 @@ export class ActionBar extends ActionRunner implements IActionRunner {
 			//this.addEmitter(item);
 			item.render(actionItemElement);
 
+			if (item instanceof BaseActionViewItem) {
```

Comment:
Would suggest adding a comment giving a brief rundown of what you put in the issue description